src/test/resources/runcmd/runs/workflow_launch.json

```diff
@@ -0,0 +1,116 @@
+{
+  "launch": {
+    "id": "3r9NghqwymK6xymExVnmdB",
+    "computeEnv": {
+      "id": "2lu3NFms1qRvwTVnrs1yhg",
+      "name": "aws_batch",
+      "description": null,
+      "platform": "aws-batch",
+      "config": {
+        "region": "eu-west-1",
+        "computeQueue": "TowerForge-2lu3NFms1qRvwTVnrs1yhg-work",
+        "dragenQueue": null,
+        "computeJobRole": null,
+        "executionRole": "arn:aws:iam::128997144437:role/TowerForge-2lu3NFms1qRvwTVnrs1yhg-ExecutionRole",
```

Comment:
Just raising a question here whether this file is necessary in the PR?

As I understand the automated cleans that Adianny has been implementing might kill this env/bucket anyhow no?

Reply:
This is just to mock the request/response of Tower API. So there is no interaction with AWS or Tower at all when running the unit tests.

Reply:
Ah, okay - thanks Jordi.